corehq/apps/app_manager/views/download.py

```diff
@@ -73,7 +75,8 @@ def download_media_suite(request, domain, app_id):
 
     """
     if not request.app.copy_of:
-        request.app.set_media_versions(None)
+        previous_version = request.app.get_latest_app(released_only=False)
```

Comment:
Hmm, you're probably right that it's safer to be consistent if there are other things already doing this.